docs/cli/oc4ids.rst

```diff
@@ -0,0 +1,123 @@
+OC4IDS Commands
+================
+
+The following commands may be used when working with `Open Contracting for Infrastructure <https://standard.open-contracting.org/infrastructure>`__ data.
+
+oc4ids-transforms
```

Comment:
I think the command should be `convert-to-oc4ids`, so that it's a verb like `compile`, `combine-release-packages`, etc.

Reply:
changed